polymesh_schema.json

```
@@ -8,6 +8,8 @@
         },
         "IdentityId": "[u8; 32]",
         "EventDid": "IdentityId",
+				"EventCounts": "Vec<u32>",
+				"ErrorAt": "(u32, DispatchError)",
```

```suggestion
        "EventDid": "IdentityId",
        "EventCounts": "Vec<u32>",
        "ErrorAt": "(u32, DispatchError)",
```
You might need to fix your editor settings to only use spaces :)